release.sh

```diff
@@ -1,7 +1,15 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
+if [[ "$BACKPORT" == "true" ]]; then
+  echo "Backport release - 'latest' tag won't be updated on docker"
+  dockerUpdateLatest="false"
```

Comment:
Correct me if I'm wrong.. This backport we will use when eg. actual version is 5.4 and we want to do some changes to < 5.4 versions?

Reply:
Exactly let's say to 4.1. Then you don't want to change someones latest do lower version...